src/components/autocomplete/js/autocompleteController.js

```diff
@@ -271,8 +271,8 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
 
     elements = {
       main:  $element[0],
-      scrollContainer: $element[0].querySelector('.md-virtual-repeat-container'),
-      scroller: $element[0].querySelector('.md-virtual-repeat-scroller'),
+      scrollContainer: $element[0].querySelector('.md-virtual-repeat-container, .md-standard-list-container'),
+      scroller: $element[0].querySelector('.md-virtual-repeat-scroller, .md-autocomplete-list-scroller'),
```

Comment:
I think that we should be consistent with `md-standard-list-*` or `.md-autocomplete-list-*`

Reply:
yah, let's go with standard then :+1: